forta-uniswap-py/src/agent_test.py

```diff
@@ -0,0 +1,185 @@
+import pytest
+from web3 import Web3
+
+from forta_agent import Finding, FindingSeverity, FindingType, create_transaction_event
+from agent import handle_transaction, get_contract_instance, ROUTER_ADDR, CONTRACT_INST
+
+
```

Comment:
Please add a comment about use of BURN_ADDR

Reply:
what comment would you like me to make